asmcli/commands/install.sh

```diff
@@ -106,6 +106,44 @@ install_canonical_controller() {
   info "...done!"
 }
 
+install_fleet_api() {
+  local PROJECT_ID; PROJECT_ID="$(context_get-option "PROJECT_ID")"
+  local CLUSTER_NAME; CLUSTER_NAME="$(context_get-option "CLUSTER_NAME")"
+  local CLUSTER_LOCATION; CLUSTER_LOCATION="$(context_get-option "CLUSTER_LOCATION")"
+  local MEMBERSHIP_NAME; MEMBERSHIP_NAME="$(generate_membership_name "${PROJECT_ID}" "${CLUSTER_LOCATION}" "${CLUSTER_NAME}")"
+
+  local FLEET_ID; FLEET_ID="$(context_get-option "FLEET_ID")"
+
+  local MEMBERSHIP_LOCATION
+  MEMBERSHIP_LOCATION="$(gcloud container fleet memberships list \
+    --project "${FLEET_ID}" \
+    --filter=name:"${MEMBERSHIP_NAME}" \
```

Comment:
In theory it's possible to have two memberships in different locations with the same name.

Reply:
Generally yes. I think the way `asmcli` handles memberships it shouldn't happen because it would generate a different name if the first name existed already in any location.